routes/user.go

```diff
@@ -410,7 +410,8 @@ func (fes *APIServer) GetHodlingsForPublicKey(
 func (fes *APIServer) GetHodlYouMap(pkid *lib.PKIDEntry, fetchProfiles bool, isDAOCoin bool, utxoView *lib.UtxoView) (
 	_youHodlMap map[string]*BalanceEntryResponse, _err error) {
 	// Get all the hodlings for this user from the db
-	entriesHodlingYou, profileHodlingYou, err := utxoView.GetHolders(pkid.PKID, fetchProfiles, isDAOCoin)
+	entriesHodlingYou, profileHodlingYou, _, _, err := utxoView.GetHolders(
```

Comment:
See comments in [here](https://github.com/deso-protocol/core/pull/1404#pullrequestreview-2313706611) about GetHolders